README.md

```diff
@@ -70,11 +70,11 @@ More information can be found [here](https://github.com/celestiaorg/celestia-nod
 `<node_type>` can be: `bridge`, `full` or `light`.
 
 ```sh
-celestia <node_type> init
+celestia <node_type> init --p2p.network=mainnet
```

Comment:
Running without p2p.network should still work, as it has the default value

Reply:
@Wondertan Update the code. We would be defaulting to DefaultNetwork; which is in-deed mainnet.